.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -29,8 +29,8 @@ If applicable, add screenshots to help explain your problem.
  - Browser [e.g. stock browser, safari]
  - Version [e.g. 59]
 
-**Were you using an @application when the bug was found?**
-  - [e.g. @buzz, @tmosphere]
+**Were you using an @‎application when the bug was found?**
```

Comment:
This undoes the invisible spaces that were added to avoid generating unwanted notifications to other GitHub users.